File: 1894-merge-strings-alternately/1894-merge-strings-alternately.ts
function mergeAlternately(word1: string, word2: string): string {
    let response = "";

    for (let i = 0; i < Math.max(word1.length, word2.length); i++) {
        if (i < word1.length) response += word1[i];
        if (i < word2.length) response += word2[i];
    }
    
    return response;
};

File: 1894-merge-strings-alternately/1894-merge-strings-alternately.py
class Solution:
    def mergeAlternately(self, word1: str, word2: str) -> str:
        merged = ""
        for i in range(0, max(len(word1), len(word2))):
            if i < len(word1):
                merged += word1[i]
            if i < len(word2):
                merged += word2[i]
        return merged